### ISSUE TITLE
reCAPTCHA: Remove icons from media item templates

### ISSUE THREAD

**Issue by machikoyasuda** [cal-itp/benefits#2538]
The main goal of this ticket is to simply remove icons and icon files from these pages of the flow:
- Eligibility Start: online account icon, contactless card icon
- Enrollment: contactless card icon
- Delete unused icon files
- Don't forget CalFresh alert box page

The icons are included in an includes like this - media-item.html - and removing this code will remove the icons from the page:
```html
    <div class="media-line">
        {% block icon %}
        {% endblock icon %}
    </div>
```

There is probably some template refactoring that could be done as a result of the removal of icons in the designs (like renaming "media-item" or perhaps getting rid of it entirely, but that doesn't necessarily have to be a part of this ticket. Giving this ticket a rating of 1.

**Comment by thekaveman** [comment]
Closed by #2564

**Comment by machikoyasuda** [comment]
I think this issue didn't get closed automatically b/c the PR got merged into a separate branch, not into `main`.
